BUILD.bazel

```diff
@@ -19,6 +19,18 @@ buildifier(
 # gazelle:go_proto_compilers //:go_apiv2
 # gazelle:go_grpc_compilers //:go_apiv2, //:go_grpc
 # gazelle:go_naming_convention import_alias
+# gazelle:resolve proto proto google/api/annotations.proto @googleapis//google/api:annotations_proto
```

Comment:
I am not sure if this is the best place for these directives.

Reply:
I will defer to @achew22 on this

Reply:
I think this is the correct place for them